<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>372</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>372</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Digit Viewer for Kaggle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>402</width>
      <height>314</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonPrevious">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/Actions-left.png</normaloff>:/Actions-left.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNext">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/Actions-right.png</normaloff>:/Actions-right.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="editGoTo">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string> - </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelUpTo">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTotalDigits">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>items</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="DigitPane" name="digit_pane" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>280</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>280</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a digit data file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset>
     <normalon>:/Actions-right.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Go to next page</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset>
     <normalon>:/Actions-left.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Go to previous page</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DigitPane</class>
   <extends>QWidget</extends>
   <header>digitpane.h</header>
   <container>1</container>
   <slots>
    <slot>onNext()</slot>
    <slot>onPrevious()</slot>
    <slot>onSetOffset(QString)</slot>
    <slot>onOffsetEdited()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNext</sender>
   <signal>triggered()</signal>
   <receiver>digit_pane</receiver>
   <slot>onNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPrevious</sender>
   <signal>clicked()</signal>
   <receiver>digit_pane</receiver>
   <slot>onPrevious()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNext</sender>
   <signal>clicked()</signal>
   <receiver>digit_pane</receiver>
   <slot>onNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrevious</sender>
   <signal>triggered()</signal>
   <receiver>digit_pane</receiver>
   <slot>onPrevious()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editGoTo</sender>
   <signal>editingFinished()</signal>
   <receiver>digit_pane</receiver>
   <slot>onOffsetEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
